ui/containers/WorkspaceDetails/components/cards/PermissionsCard.tsx

```diff
@@ -10,6 +10,7 @@ interface Props {
   members?: Member[];
   memberLoading: boolean;
   addMemberDisabled: boolean;
+  columnLoader: boolean;
```

Comment:
@shafi1379 we need to use following naming conventions here. @afoerster , @shafi1379 , how about naming it as `isColumnLoading` ? or something like `isLoading`?

Reply:
Yeah agree `isColumnLoading` is a better description

Reply:
or `isLoading` since it's not a column but the whole block we are putting the spinner on. Either way

Reply:
@afoerster , the lines you commented is outdated version. we are no longer using `columnHeader`. we are using `memberLoading`.